### ISSUE TITLE
Language bindings

### ISSUE THREAD

**Issue by ben-white-openx** [google/zetasketch#14]
Is this the implementation referenced here in the BigQuery docs?

https://cloud.google.com/bigquery/docs/sketches#sketch_integration

If so, should this be linked, and are there other language bindings coming?

**Comment by zfraa** [comment]
Yes it is! The implementation here produces bitwise identical sketches to the library backing the BigQuery HLL_COUNT.* functions. 

No open sourcing of other language bindings is planned at the moment. 

We'll iterate with the authors of https://cloud.google.com/bigquery/docs/sketches#sketch_integration.
